site/docs/windows.md

```diff
@@ -184,7 +184,14 @@ To build C++ targets, you need:
     SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk).
 
     The Windows SDK contains header files and libraries you need when building
-    Windows applications, including Bazel itself.
+    Windows applications, including Bazel itself. You can specify Windows SDK version
```

Comment:
I'd start with describing the default case, i.e. when `BAZEL_WINSDK_FULL_VERSION` is not specified.

Then you can elaborate what if there are multiple SDK versions installed, i.e. when the user may need to set `BAZEL_WINSDK_FULL_VERSION`.

Reply:
Thanks, done!